packages/addons/service/proftpd/source/etc/proftpd.conf

```diff
@@ -0,0 +1,60 @@
+ServerName "ProFTPD on LibreELEC"
```

Comment:
If idiots are going to expose FTP servers to the interwebs it's better to remove visible references to LE that might help an attacker profile a target.